requirements-build-docs.txt

```diff
@@ -1,5 +1,4 @@
 numpy
-pandas
```

Comment:
I'm surprised to see a removal of pandas for doc building. We are using it in examples, eg
02_decoding/plot_haxby_anova_svm.py

Reply:
Somehow it was listed twice previously -- this only removes the duplication. It's still included in the requirements !

Reply:
Ah, the problem of looking only at the diff.